src/pages/DeliveryModeScreen/DeliveryModeScreen.tsx

```diff
@@ -0,0 +1,39 @@
+import { useContext } from 'react';
+import { Form } from 'react-final-form';
+import Button from '@mui/material/Button';
+import MenuItem from '@mui/material/MenuItem';
+import { Select } from 'mui-rff';
+import FormScreens from 'constants/formScreens';
+import WizardFormContext from 'contexts/WizardFormContext';
+import { validateIsRequired } from 'utils/validation';
+
+type DeliveryMode = { deliveryMode: null };
+
+export default function DeliveryModeScreen() {
+  const { onSaveFormValues } = useContext(WizardFormContext);
+
+  return (
+    <Form<DeliveryMode>
+      onSubmit={(values) => {
+        onSaveFormValues(FormScreens.DELIVERY_MODE, values);
+      }}
+      render={({ handleSubmit }) => (
+        <form onSubmit={handleSubmit}>
+          <Select
+            fieldProps={{ validate: validateIsRequired }}
+            label='Delivery Type'
+            name='deliveryType'>
+            <MenuItem value='postOffice'>Post Office Delivery</MenuItem>
```

Comment:
Save values to constant. They will be reused
Also save options to another const as well